azerothcore/azerothcore-wotlk
# fix(DB/Dungeon): improved Upper Blackrock

Changes:

Min, Max & Modifier Damage for whole Dungeon.
Added flags for each boss, so they will appear with skull instead regular level elite.

All checks have been done trough Classic Realms and Youtube videos.

How to test:
Apply SQL
Restart Server and delete your Cache.
Make sure to add very very basic gear towards Dungeon ilevel.

Before you start checking damage:
.mod hp 50000
.learn 1908 (Uber heal over time)

If there is any issue with damage and if you get instantly killed, lemme know.

PS. Last Dungeon (Blackrock Depths) will be completed during next weekend and it will be last one for Classic.
After that, i will start working on Lich King Dungeons.
